docs/data/charts/tooltip/tooltip.md

```diff
@@ -7,8 +7,66 @@ title: Charts - Tooltip
 
 <p class="description">Tooltip provides extra data on charts item.</p>
 
+In all charts components, you can pass props to the tooltip by using `tooltip={{...}}`.
+If you are using composition, you can add the `<Tooltip />` component and pass props directly.
+
 ## Interactions
 
+The tooltip can be triggered by two kind of events:
+
+- `'item'` which means user mouse is hover an item of the chart. The tooltip will display data about this specific items.
+- `'axis'` which means user mouse position is associated to a value of the x-axis. The tooltip will display data about all series at this specific x value.
+
+{{"demo": "Interaction.js", "bg": "inline"}}
+
 ## Highlights
 
-## Rendered Tooltip
+## Highlighting axis
+
+You can hightligh data based on mouse position.
+By default those highlights are lines, but is can also be a vertical band if your x-axis use `scaleType: 'band`.
+
+On chart, to customize this behavior,you can use
+
+```jsx
+highlight={{
+    x: 'line', // Or 'none', or 'band'
+    y: 'line', // Or 'none'
+    }}
+```
+
+{{"demo": "BandHighlight.js",  "bg": "inline"}}
+
+## Highlighting series 🚧
+
+When the tooltip is highlighting elements, their style can be modified, and the style of other elements can be fade-out.
+
+This is under construction, and require some management to avoid ending up with a blinking christmas tree.
+
+Here is a demo of what it could be but needing much more work.
+
+{{"demo": "Highlights.js", "bg": "inline"}}
+
+## Customization
+
+The format of data rendered in the tooltip can be modified thanks to series' `valueFormatter` property.
+Same applied for x values when tooltip is triggered by `'axis'`.
+
+Here is a demo with:
+
+- Time axis formatted to only show the year
+- Number values formatted in U.S. Dollar.
+
+{{"demo": "Formatting.js", "bg": "inline"}}
+
+## Composition
+
+If you're using composition, by default the mouse event will not be listen.
+If you need it (for example to support highlight or tooltip with `trigger='axis'`) you should add `<AxisInteractionListener />`.
+
+```jsx
+<ChartContainer {...}>
+  <AxisInteractionListener listen/>
```

Comment:
What's the reasoning behind this limitation?
It looks like something people could easily miss

Reply:
The hook listening to the mouse move needs to be inside `ChartContainer` to get the providers. And to know if it worths listening to them, I need access to `tooltip` and `highlights` settings.

I created a distinct component such that in s charts (like `<BarChart />`) I could compute `listen` without having to pass `tooltip` and `highlights` settings to the `ChartContainers`

A better solution could be to introduce `disableAxisListener`  to the `<ChartContainers />` such that charts components like `<BarChart />` could still have smart behavior, and dev using composition can not miss it since it's `true` by default

Reply:
> A better solution could be to introduce `disableAxisListener` to the `<ChartContainers />` such that charts components like `<BarChart />` could still have smart behavior, and dev using composition can not miss it since it's `true` by default

This direction sounds way less error-prone and should provide a bit better DX. 🤔 👍